eng/Version.Details.xml

```diff
@@ -1,64 +1,64 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Extensions.CommandLineUtils.Sources" Version="5.0.0-alpha1.19517.7">
+    <Dependency Name="Microsoft.Extensions.CommandLineUtils.Sources" Version="5.0.0-alpha1.19518.28">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>f15f3278c73c72a6546d7cb2c1bd54a541ffc83e</Sha>
+      <Sha>02c5fce040a75bcc3670de5eb6e92b568a294e16</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.HashCodeCombiner.Sources" Version="5.0.0-alpha1.19517.7">
+    <Dependency Name="Microsoft.Extensions.HashCodeCombiner.Sources" Version="5.0.0-alpha1.19518.28">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>f15f3278c73c72a6546d7cb2c1bd54a541ffc83e</Sha>
+      <Sha>02c5fce040a75bcc3670de5eb6e92b568a294e16</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.NonCapturingTimer.Sources" Version="5.0.0-alpha1.19517.7">
+    <Dependency Name="Microsoft.Extensions.NonCapturingTimer.Sources" Version="5.0.0-alpha1.19518.28">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>f15f3278c73c72a6546d7cb2c1bd54a541ffc83e</Sha>
+      <Sha>02c5fce040a75bcc3670de5eb6e92b568a294e16</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="5.0.0-alpha1.19517.7">
+    <Dependency Name="Microsoft.Extensions.Logging" Version="5.0.0-alpha1.19518.28">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>f15f3278c73c72a6546d7cb2c1bd54a541ffc83e</Sha>
+      <Sha>02c5fce040a75bcc3670de5eb6e92b568a294e16</Sha>
     </Dependency>
     <Dependency Name="System.Diagnostics.DiagnosticSource" Version="4.6.0" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
@wtgodbe @mmitche is System.Diagnostics.DiagnosticSource no longer produced in CoreFx?

Reply:
Nope, I see a 5.0.0-alpha1.19520.7 produced yesterday

Reply:
It updated in extensions. It looks like a bug.  I'll investigate right now: https://github.com/dotnet/arcade/issues/4155